BookstoreDatabase/Data/BookstoreDatabaseDesignTimeFactory.cs

```diff
@@ -7,22 +7,23 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace BookstoreDatabase.Data
 {
-    internal class BookstoreDatabaseDesignTimeFactory : IDesignTimeDbContextFactory<DBContext>
+    internal class BookstoreDatabaseDesignTimeFactory : IDesignTimeDbContextFactory<BookstoreContext>
     {
-        public DBContext CreateDbContext(string[] args)
+        public BookstoreContext CreateDbContext(string[] args)
         {
             var configuration = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.json")
                 .Build();
 
-            var optionsBuilder = new DbContextOptionsBuilder<DbContext>();
```

Comment:
Changed the name from DBContext to BookstoreContext to try and make it easier to find what was causing issues with creating the database. DbContext on this line needed to be of the type we created, so for this to work it needed to be DBContext between the <>